docs/source/app_developers_guide/environment_setup.rst

```diff
@@ -30,54 +30,31 @@ The following tools are required:
 * `Docker Compose <https://docs.docker.com/compose/install/>`_ (Linux only)
 
 
-Step One: Clone Repository
---------------------------
+Get the Docker Yaml File
```

Comment:
This is being changed by another PR. I'll rebase this one when that gets merged today.